composer.json

```
@@ -9,7 +9,10 @@
 		}
 	],
 	"require": {
-		"php": "^8.0"
+		"php": "^8.0",
+		"baraja-core/plugin-system": "^v2.2.0",
```

Verzi dávej relativně, aby šlo automaticky aktualizovat. Fixuje se to na první setinku:

```suggestion
		"baraja-core/plugin-system": "^2.2",
```